template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Extensions/AuthenticationServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Authentication.Extensions
+{
+    public static class AuthenticationServiceCollectionExtensions
+    {
+        public static IServiceCollection AddWebApplicationAuthentication(
+            this IServiceCollection services,
+            Action<AuthenticationOptions> configureAuthentication = null,
```

Comment:
Is it worth the additional optional parameters, when they can just call Configure beforehand?